agents/Group23/mcts.py

```diff
@@ -130,6 +133,13 @@ def get_heuristic_moves(self, node: TreeNode) -> list[Move]:
         https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=4406406
         """
         moves = node.moves
+        print('======================================')
+        print(f'Retrieved {len(moves)} moves from the node.')
+        print(f'Moves:')
+        for move in moves:
+            print(f' - ({move.x}, {move.y}): priority {move.priority}')
```

Comment:
Should we remove these print statements, or are they still required for debugging?

Reply:
Yeah that's a good point. I've got a commit on the `submission-setup` branch that tidies up all the print statements and such so no need to worry about it for this one.

Reply:
Cool, sounds good :)